OpenLIFUData/OpenLIFUData.py

```diff
@@ -73,6 +74,53 @@ class OpenLIFUDataParameterNode:
     loaded_transducers : "Dict[str,SlicerOpenLIFUTransducer]"
     loaded_plan : "Optional[SlicerOpenLIFUPlan]"
 
+
+class AddNewSubjectDialog(qt.QDialog):
+    """ Add new subject dialog """
+
+    def __init__(self, database, parent="mainWindow"):
```

Comment:
oh, also now you should be able to remove `database` as an argument here!

Reply:
thanks for catching this!